.github/workflows/runtests.yml

```diff
@@ -148,8 +148,8 @@ jobs:
       - run: python -m pip install --upgrade pip
       - run: pip install mpmath gmpy2 matplotlib numpy scipy aesara ipython \
                          symengine cython llvmlite wurlitzer \
-                         autowrap numexpr 'antlr4-python3-runtime==4.7.*'
-      - run: pip install numba
+                         autowrap numexpr 'antlr4-python3-runtime==4.7.*' \ 
+                         pip install numba
```

Comment:
```suggestion
                         numba
```

Reply:
These are package names. We do not want to install `pip` (as it is already installed) and not install `install` (as it is not needed).